Source/WebGPU/WGSL/WGSL.h

```diff
@@ -143,6 +145,9 @@ struct BindGroupLayoutEntry {
     OptionSet<ShaderStage> visibility;
     using BindingMember = std::variant<BufferBindingLayout, SamplerBindingLayout, TextureBindingLayout, StorageTextureBindingLayout, ExternalTextureBindingLayout>;
     BindingMember bindingMember;
+    std::optional<uint32_t> vertexBufferDynamicOffset;
```

Comment:
@tadeuzagallo these are byte offsets. I could encode as uint32_t sized offsets instead, it would save some divisions by sizeof(uint32_t) most likely

Reply:
The offset is cumulative from the first BindGroupLayoutEntry.

So the vertex offset of the BindGroupLayoutEntry of the nth BindGroupLayoutEntry in a pipeline is really:

`* vertexBufferDynamicOffset + totalVertexOffsetUpTo(n)` where `totalVertexOffsetUpTo` returns the total cumulative vertex offsets of the preceding BindGroupLayoutEntry instances